modules/core/shared/src/main/scala/lucuma/core/math/units.scala

```diff
@@ -183,6 +184,10 @@ trait units {
 
   implicit val convMP: UnitConverter[PosInt, Micrometer, PosInt, Picometer] =
     unsafePosIntConverter[Micrometer, Picometer]
+
+  // Integer Percents
+  type ZeroTo100  = numeric.Interval.Closed[0, 100]
+  type IntPercent = Int Refined ZeroTo100
```

Comment:
This doesn't really have units. I started out with it having the coulomb `Percent` unit, but all it seemed to buy me was tears. I want to share it between the API and the UI, but maybe there is a better place than `math.units`?

Reply:
Do we ever expect decimals here? maybe `spire.Rational` could be an alternative if so

Reply:
Andy settled on integer percents. So, for 3 equal partners, the PI will need to decide who to ask 34% from.